exercises/aux_files.md

```diff
@@ -0,0 +1,133 @@
+# Exercises: Using aux files
+
+We want to explore auxiliary files in these exercises, which allow you to load up contextual information
+alongside your data. 
+
+## 1. Aux files with a plain text file
+
+In this example we will add a simple linear trend to ten years of data, and view the results using
```

Comment:
"to ten years of data". This would only work if the participants have 10 years of data with monthly frequency. Can we provide such a mock dataset?

Reply:
yes: #33